cnf-certification-test/networking/suite.go

```diff
@@ -153,16 +156,16 @@ func testMultusNetworkConnectivity(env *provider.TestEnvironment, count int, aIP
 	for _, put := range env.Pods {
 		// The first container is used to get the network namespace
 		aContainerInPod := &put.Spec.Containers[0]
-
-		if _, ok := env.SkipNetTests[put]; ok {
+		testpod := provider.PodStruct(*put)
```

Comment:
This copy operation creates a new address which will not be matched in the skip object